hack/lib/test.sh

```diff
@@ -75,16 +77,16 @@ kube::test::object_assert() {
   local expected=$4
   local args=${5:-}
 
-  for j in $(seq 1 ${tries}); do
-    res=$(eval kubectl get "${kube_flags[@]}" ${args} ${object} -o go-template=\"${request}\")
+  for j in $(seq 1 "${tries}"); do
+    res=$(eval kubectl get "${kube_flags[@]}" "${args}" "${object}" -o go-template=\""${request}"\")
```

Comment:
Fix for this:

```
In ./hack/lib/test.sh line 78:
  for j in $(seq 1 ${tries}); do
                   ^------^ SC2086: Double quote to prevent globbing and word splitting.


In ./hack/lib/test.sh line 79:
    res=$(eval kubectl get "${kube_flags[@]}" ${args} ${object} -o go-template=\"${request}\")
                                              ^-----^ SC2086: Double quote to prevent globbing and word splitting.
                                                      ^-------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                                 ^--------^ SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
is args depending on splitting though?

Reply:
Yep, I think you're right, will revert and set a directive to ignore